bitrise.yml

```diff
@@ -1614,8 +1614,16 @@ workflows:
     - 2_certificate_and_profile
     - 3_provisioning_and_npm_installation
     steps:
-    - xcode-test@6.0.1:
-        timeout: 3000
+    - xcode-build-for-test@3:
+        timeout: 600
```

Comment:
should this be `6000`?

Reply:
I separated the build and test step to see if Xcode 26 actually takes too long to build.

Reply:
The build usually finishes in 3 min.

Reply:
okay nice! thanks for setting this up!